scripts/aws/staging-deployment.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -e
+
+if env | grep -q "MMW_DEPLOY_DEBUG"; then
+  set -x
+fi
+
+CURRENT_STACK_COLOR=$(aws cloudformation describe-stacks \
+  --profile mmw-stg \
```

Comment:
Fixed in https://github.com/WikiWatershed/model-my-watershed/commit/0d839e798339d083f773c999819b886afc805ec3.